### PR TITLE
Create google_cloud_speech.py

### PR DESCRIPTION
**Pull request by GrahamDerry**
I tried to make a script to use Google Cloud Speech to transcribe text. 
Google cloud speech requires credentials. 
I created a service account on the Google Cloud Platform, and downloaded a key in the form of a JSON file.
I don't know how to point to that key file in my script, line 14

### CODE REVIEW

**Comment by GrahamDerry** [comment]
https://cloud.google.com/speech-to-text/docs/troubleshooting

I think I'm having trouble with this step 
"You must have a service account for your project, download the key (JSON file) for your service account to your development environment, and then set the location of that JSON file to an environment variable named GOOGLE_APPLICATION_CREDENTIALS."

File: google_cloud_speech.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+r = sr.Recognizer()
+
+# Create a microphone objec that gets system default
+mic = sr.Microphone()
+
+# Listen for audio and save it
+with mic as source:
+    audio = r.listen(source)
+
+GOOGLE_CLOUD_SPEECH_CREDENTIALS = /Users/grahamderry/praxis-road-272622-d0ad9ed1867c.json
```
Comment (max-carey) [inline google_cloud_speech.py:14]: The path to the JSON file probably needs to be in quotes.

Comment (max-carey) [reply google_cloud_speech.py:14]: Actually it says the path to the JSON file needs to be in an environment variable. An environment variable is something you usually set in the terminal (which has to be set every time you want to run the script and open a new terminal. In the terminal you type `export GOOGLE_CLOUD_SPEECH_CREDENTIALS='<MY_PATH>'`

Comment (max-carey) [reply google_cloud_speech.py:14]: Another could thing to do is create a file called app-env (or whatever) in which you have a bunch of one liners like the one above (for multiple things in environment variables). Then you simply `source app-env` every time before running your script.

**Review by max-carey — APPROVED** [review]

**Pull request merged**